packages/discord.js/src/structures/Guild.js

```diff
@@ -697,15 +697,16 @@ class Guild extends AnonymousGuild {
    * Data for the Guild Widget Settings object
    * @typedef {Object} GuildWidgetSettings
    * @property {boolean} enabled Whether the widget is enabled
-   * @property {?(TextChannel|NewsChannel|VoiceChannel|StageChannel|ForumChannel|MediaChannel)} channel
+   * @property {?(TextChannel|AnnouncementChannel|VoiceChannel|StageChannel|ForumChannel|MediaChannel)} channel
    * The widget invite channel
    */
 
   /**
    * The Guild Widget Settings object
    * @typedef {Object} GuildWidgetSettingsData
    * @property {boolean} enabled Whether the widget is enabled
-   * @property {?(TextChannel|NewsChannel|VoiceChannel|StageChannel|ForumChannel|MediaChannel|Snowflake)} channel
+   * @property {?(TextChannel|AnnouncementChannel|VoiceChannel|StageChannel|ForumChannel|
```

Comment:
Double check the docs don't break with this going multiline

Reply:
Looks good 
![Screenshot 2024-10-06 224951](https://github.com/user-attachments/assets/f55c49ca-1c90-4258-abcb-e1af51ebecb9)

Perhaps I should disable the rule for this one, idk if it looks okay in the JSDoc itself

Reply:
This is already done in plenty of places.

Reply:
resolving, this is ok IMO.